.byebug_history

```diff
@@ -0,0 +1 @@
+continue
```

Comment:
Make sure to delete this file. it's created when debugging and does not need to be committed.